From b7ec8a024d60e7570b172f33a67ad8238f48c26d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Steven!=20Ragnar=C3=B6k?= <steven@nuclearsandwich.com>
Date: Fri, 16 Nov 2018 15:52:51 -0800
Subject: [PATCH] Add Suggests for vendor typesupport packages.

---
 debian/control.em | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/control.em b/debian/control.em
index 7cbf65a..3a59fb9 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -11,4 +11,5 @@ Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends))
 @[if Conflicts]Conflicts: @(', '.join(Conflicts))@\n@[end if]@
 @[if Replaces]Replaces: @(', '.join(Replaces))@\n@[end if]@
+Suggests: ros-dashing-rosidl-typesupport-connext-c, ros-dashing-rosidl-typesupport-fastrtps-c, ros-dashing-rosidl-typesupport-opensplice-c
 Description: @(Description)
-- 
2.7.4

