From 31ae634503e0be95594f717fa28486d5c28f03a0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Steven!=20Ragnar=C3=B6k?= <steven@nuclearsandwich.com>
Date: Fri, 16 Nov 2018 15:55:12 -0800
Subject: [PATCH] Add Suggests for vendor typesupport packages.

Signed-off-by: Michael Carroll <michael@openrobotics.org>
---
 debian/control.em | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/control.em b/debian/control.em
index 7cbf65a..36f2685 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -11,4 +11,5 @@ Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends))
 @[if Conflicts]Conflicts: @(', '.join(Conflicts))@\n@[end if]@
 @[if Replaces]Replaces: @(', '.join(Replaces))@\n@[end if]@
+Suggests: ros-eloquent-rosidl-typesupport-connext-cpp, ros-eloquent-rosidl-typesupport-fastrtps-cpp, ros-eloquent-rosidl-typesupport-opensplice-cpp
 Description: @(Description)
-- 
2.17.1

