From b4f63255948967c220d7259f30cbb05ecb0c1dea Mon Sep 17 00:00:00 2001
From: Jacob Perron <jacob@openrobotics.org>
Date: Fri, 24 Apr 2020 23:54:56 -0700
Subject: [PATCH] Add Suggests for vendor typesupport packages

Signed-off-by: Jacob Perron <jacob@openrobotics.org>
---
 debian/control.em | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/control.em b/debian/control.em
index 6d7b65c..e9d60d3 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -11,4 +11,5 @@ Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends))
 @[if Conflicts]Conflicts: @(', '.join(Conflicts))@\n@[end if]@
 @[if Replaces]Replaces: @(', '.join(Replaces))@\n@[end if]@
+Suggests: ros-rolling-rosidl-typesupport-connext-cpp, ros-rolling-rosidl-typesupport-fastrtps-cpp
 Description: @(Description)
-- 
2.29.2

